cmd/broker/provisioning_test.go

```diff
@@ -351,7 +351,7 @@ func TestProvisioning_Preview(t *testing.T) {
 		}`)
 	opID := suite.DecodeOperationID(resp)
 
-	suite.processProvisioningByOperationID(opID)
+	suite.waitForRuntimeAndMakeItReady(opID)
```

Comment:
Preview is now configured to not use Provisioner, but only Runtime REsource (KIM)